pkg/executables/helm.go

```diff
@@ -76,7 +55,7 @@ func (h *Helm) Template(ctx context.Context, ociURI, version, namespace string,
 	params = h.addInsecureFlagIfProvided(params)
 	params = append(params, "-f", "-")
 
-	result, err := h.executable.Command(ctx, params...).WithStdIn(valuesYaml).WithEnvVars(h.env).Run()
+	result, err := h.executable.Command(ctx, params...).WithStdIn(valuesYaml).WithEnvVars(h.helmOptions.Env).Run()
```

Comment:
imo env is an option that should be part of this helm struct and not the helm.Options, since it only applies to this particular client implementation

Reply:
Is this even being used?

Reply:
Discussed offline. We'll make it a bit more implementation agnostic by using a `helm.WithProxyConfig` and then so the executable will turn the struct into `env`